corehq/form_processor/backends/sql/ledger.py

```diff
@@ -77,6 +77,7 @@ def _lazy_original_balance():
         ledger_value.track_create(transaction)
         # only do this after we've created the transaction otherwise we'll get the wrong delta
         ledger_value.balance = new_ledger_values.balance
+        ledger_value.last_modified = stock_trans.timestamp
```

Comment:
Does this mean that `ledger_value.last_modified` is changing from representing approximate submit time (off by a few msec-to-sec) to representing completion time? (And if so, just checking that's desired.)